src/Microsoft.AspNet.Razor.Runtime/TagHelpers/OutputElementHintAttribute.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Framework.Internal;
+
+namespace Microsoft.AspNet.Razor.Runtime.TagHelpers
+{
+    /// <summary>
+    /// Provides a hint of the <see cref="ITagHelper"/>'s output element.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class, AllowMultiple = false, Inherited = false)]
```

Comment:
Decided to not have the attribute inheritable since `TargetElement` is not inheritable.